src/plugins/kibana_utils/common/index.ts

```diff
@@ -22,9 +22,11 @@ export * from './of';
 export * from './ui';
 export * from './state_containers';
 export * from './typed_json';
+export * from './errors';
 export { createGetterSetter, Get, Set } from './create_getter_setter';
 export { distinctUntilChangedWithInitialValue } from './distinct_until_changed_with_initial_value';
 export { url } from './url';
 export { now } from './now';
 export { calculateObjectHash } from './calculate_object_hash';
 export { defaultFeedbackMessage } from './default_feedback_message';
+export * from './field_mapping';
```

Comment:
Looks like the contents of `field_mapping` is only used in `data` and `saved_objects`. Considering that `saved_objects` already has a dependency on `data`, any thoughts on moving these items to data instead?

This would help get rid of the [circular dependency with data](https://github.com/elastic/kibana/blob/240371af9235687f8d64c4a2647892c8a8cf1f4a/src/plugins/kibana_utils/common/field_mapping/mapping_setup.ts#L23-L24) which we are going to need to get rid of eventually anyway.